README.md

```diff
@@ -5,57 +5,120 @@ Pulp 3 Ansible installer.
 be cloned.
 
-Usage
------
-
-The ansible-pulp3 repository contains a playbook, group vars, host vars, and so
-on. However, these files are used by the test harness, and *must not* be relied
-upon. To reliably run the Ansible Pulp 3 installer, create your own playbook,
-group vars, host vars, and so on.
-
-For example, let's say you have virtual machines with hostnames
-`centos-7-pulp-3`, `fedora-27-pulp-3`, and `fedora-28-pulp-3`. Let's also say
-that you've configured passwordless SSH, and that the remote Ansible users have
-passwordless sudo access. You can install Pulp 3 on them with the following
-script:
-
-```bash
-cat >site.yml <<EOF
-- hosts: all
-  roles:
-    - pulp3-postgresql
-    - pulp3-workers
-    - pulp3-resource-manager
-    - pulp3-webserver
-EOF
-cat >inventory <<EOF
-centos-7-pulp-3
-fedora-27-pulp-3
-fedora-28-pulp-3
-EOF
-mkdir {group,host}_vars
-cat >group_vars/all <<EOF
-pulp_secret_key: secret
-pulp_default_admin_password: password
-EOF
-cat >host_vars/fedora-27-pulp-3 <<EOF
-ansible_python_interpreter: /usr/bin/python3
-EOF
-
-# The current ANXS.postgresql release doesn't support Fedora.
-cat >requirements.yml <<EOF
-- src: https://github.com/ANXS/postgresql
-  version: master
-  name: ANXS.postgresql
-EOF
+Requirements
+------------
+
+The [control node](https://docs.ansible.com/ansible/2.5/network/getting_started/basic_concepts.html#control-node)
+must have Python 3 and Ansible installed.
+
+The [managed node](https://docs.ansible.com/ansible/2.5/network/getting_started/basic_concepts.html#managed-nodes)
+must be one of these currently supported operating systems:
+* Fedora 27
+* Fedora 28
+* CentOS 7
+
+Knowledge of [Ansible](https://www.ansible.com/) is not necessary for the
+[Vagrant](https://www.vagrantup.com/) installed, but will be very helpful for
+installations that go beyond the default configuration.
+
+User Installation
+-----------------
+
+These roles can be used against any managed node and are highly configurable.
+Knowledge of [ansible basics](https://docs.ansible.com/ansible/2.5/user_guide/intro_getting_started.html).
+will be helpful, but even if you are new to Ansible, this section will get you
+started, or you can try the Vagrant installations to bypass the Ansible boilerplate.
+
+First, you will need to configure ssh between your control node and your
+managed node. When you can ssh into the managed node without a password, you
+are ready to move to the next step.
+
+Next, add the managed node's hostname or ip address to `/etc/ansible/hosts`.
+
+It may be helpful to ensure that Ansible can communicate with the managed node.
+
+```
+ansible all -m ping -u <managed_node_username>
+```
+
+You should now be able to run the example playbook. The playbook uses `become`
+(root), so when prompted, you will need to provide the password for the managed
+node user.
+
+```
 ansible-galaxy install -r requirements.yml
+ansible-playbook user-sandbox.yml -u <managed_node_username> --ask-become-pass
```

Comment:
you should mention what this user-sandbox.yml file is. it's not clear where i am supposed to put all the configuration parameters for the install.

Reply:
Added some explanation below.